# Description: fix hba declaration to match get_action_area return type
#  Gtk::Dialog 2.18 now defines this as a generic ButtonBox
# Forwarded: no
# Bug-Debian: 577371
# Author: Damyan Ivanov <dmn@debian.org>
--- a/src/workhelper.cpp
+++ b/src/workhelper.cpp
@@ -39,7 +39,7 @@ WorkHelper::WorkHelper()
 	//set_resize_mode(Gtk::RESIZE_PARENT);
 	set_reallocate_redraws(true);
 
-	Gtk::HButtonBox *hba = get_action_area();
+	Gtk::ButtonBox *hba = get_action_area();
 
 	add_events(Gdk::ENTER_NOTIFY_MASK | Gdk::LEAVE_NOTIFY_MASK);
 	
